cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -178,18 +178,25 @@ private ShesmuSequencing makeShesmuSequencing(List<Sample> samples, MetricCatego
     boolean hasWaiting = false;
     for (Sample sample : samples) {
 
-      if (sample.getDataReviewPassed() == null || sample.getQcPassed() == null) {
+      if (sample.getDataReviewPassed() == null || (sample.getQcPassed() == null && sample.getQcUser() == null)) {
         hasWaiting = true;
       } else if (sample.getDataReviewPassed() && sample.getQcPassed()) {
         hasPassed = true;
       }
 
-      if (!(sample.getRun() == null && type.equals(MetricCategory.LIBRARY_QUALIFICATION))) {
-        shesmuSamples.add(new ShesmuSample.Builder()
-            .id(sample.getId())
-            .supplemental(!Objects.equals(sample.getRequisitionId(), reqId))
-            .qcFailed(getQcFailed(sample))
-            .build());
+      if (sample.getRun() != null) {
+        Run currRun = sample.getRun();
+        if (currRun.getDataReviewPassed() == null || (currRun.getQcPassed() == null && currRun.getQcUser() == null)){
+          hasWaiting = true;
+        }
+
+        if (!type.equals(MetricCategory.LIBRARY_QUALIFICATION)) {
```

Comment:
This check should be removed. The samples that were previously being excluded here were those that were `LIBRARY_QUALIFICATION` _and_ had no run. Library qualification samples with a run should still be included, and excluding samples with no run is sufficient - the check on `MetricCategory` isn't necessary